index.d.ts

```diff
@@ -0,0 +1,214 @@
+import * as ObisNames from './lib/ObisNames';
+import {ObisMeasurement} from './lib/ObisMeasurement';
+
+export type ObisOptions = ObisBaseOptions & ObisProtocolOptions & ObisTransportOptions;
+export type ObisLanguage = 'en' | 'de';
+
+
```

Comment:
All these exports are missing here:
https://github.com/Apollon77/smartmeter-obis/blob/d7918fb29b48a8ad3b2db7f1ebefda48f3a809db/index.js#L78-L87

Reply:
Ah yes, thanks for mentioning `SmlUnits` and `ObisMeasurement`!

For the `Transport` classes however, I decided to implement an abstract `ObisTransport` class in `index.d.ts`. The motivation was, that all of these functions have the same interface in common and have no further attributes that should be accessible from the outside.

Regarding the `Protocol` classes, I'm honestly not quite sure for what reason they are part of the original `index.js` exports. Their single occurrence is capsuled inside the `init` function, and there's no way to access or modify them from outside the library. Therefore, I found it more confusing than helpful adding these exports.